docs/en/latest/how-to-build.md

```diff
@@ -119,7 +119,11 @@ Follow the steps below to install Apache APISIX via the source release package.
 
   **Note**: If `make deps` fails with "install `lualdap` failed" with an error like `Could not find header file for LDAP` try the solution below.
 
-  Solution: Set `LDAP_DIR` with `luarocks config` manually. For example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`.
+      Solution: luarocks supports adding custom package addresses to the project. You can set `LDAP_DIR` with `luarocks config` manually, for example `luarocks config variables.LDAP_DIR /usr/local/opt/openldap/`. You should config as follows:
+      1. Install `openldap` locally using tools including but not limited to 'brew';
```

Comment:
Since you mention the `brew`, let's also give an example to use `brew` to install openldap.

Reply:
yeah, I think my document now is better.

Reply:
@Chever-John I think, the conversation should be marked as resolved by the reviewer, but not you